aws_cloudtrail_rules/aws_iam_entity_created_without_cloudformation.yml

```diff
@@ -106,6 +106,48 @@ Tests:
           "eventType": "AwsApiCall",
           "recipientAccountId": "123456789012"
       }
+  -
+    Name: IAM Entity Created Manually With Approved Role Pattern
+    ExpectedResult: true
```

Comment:
Should this be false?

Reply:
Yup, debugged it to find that I needed the `invokedBy: "cloudformation.amazonaws.com"` in the example, added that and the test started working as intended